eradiate/scenes/atmosphere/tests/test_homogeneous.py

```diff
@@ -37,43 +37,40 @@ def test_rayleigh_homogeneous(mode_mono, ref):
     assert kernel_dict.load() is not None
 
     # Construct with parameters
-    r = RayleighHomogeneousAtmosphere(sigma_s=1e-5)
-    assert r.sigma_s == ureg.Quantity(1e-5, ureg.m ** -1)
+    r = HomogeneousAtmosphere(sigma_s=1e-5)
+    assert r.sigma_s.value == ureg.Quantity(1e-5, ureg.m ** -1)
 
     eradiate.set_mode("mono", wavelength=650.)
-    r = RayleighHomogeneousAtmosphere(height=ureg.Quantity(10, ureg.km))
+    r = HomogeneousAtmosphere(height=ureg.Quantity(10, ureg.km))
     assert r.height == ureg.Quantity(10, ureg.km)
 
-    # check if sigma_s was correctly computed using the mode wavelength value
+    # Check if sigma_s was correctly computed using the mode wavelength value
     wavelength = eradiate.mode.wavelength
-    assert np.isclose(r._sigma_s, compute_sigma_s_air(wavelength=wavelength))
+    assert np.isclose(r._sigma_s.value, compute_sigma_s_air(wavelength=wavelength))
 
-    # check if automatic scene width works as intended
+    # Check if automatic scene width works as intended
     assert np.isclose(r.kernel_width, 10. / compute_sigma_s_air(wavelength=wavelength))
 
     # Check if produced scene can be instantiated
     assert KernelDict.empty().add(r).load() is not None
 
     # Check that sigma_s wavelength specification is correctly taken from eradiate mode
     eradiate.set_mode("mono", wavelength=650.)
-    r = RayleighHomogeneousAtmosphere(sigma_s="auto")
-    assert r._sigma_s != compute_sigma_s_air(wavelength=550.)
+    r = HomogeneousAtmosphere(sigma_s="auto")
+    assert np.allclose(r._sigma_s.value, compute_sigma_s_air(wavelength=650.))
 
     # Check that attributes wrong units or invalid values raise an error
     with pytest.raises(UnitsError):
-        RayleighHomogeneousAtmosphere(height=ureg.Quantity(10, "second"))
+        HomogeneousAtmosphere(height=ureg.Quantity(10, "second"))
 
     with pytest.raises(UnitsError):
-        RayleighHomogeneousAtmosphere(width=ureg.Quantity(5, "m^2"))
+        HomogeneousAtmosphere(width=ureg.Quantity(5, "m^2"))
 
     with pytest.raises(UnitsError):
-        RayleighHomogeneousAtmosphere(sigma_s=ureg.Quantity(1e-7, "m"))
+        HomogeneousAtmosphere(sigma_s=ureg.Quantity(1e-7, "m"))
 
     with pytest.raises(ValueError):
-        RayleighHomogeneousAtmosphere(height=-100.)
+        HomogeneousAtmosphere(height=-100.)
 
     with pytest.raises(ValueError):
-        RayleighHomogeneousAtmosphere(width=-50.)
-
-    with pytest.raises(ValueError):
-        RayleighHomogeneousAtmosphere(sigma_s=-1e-7)
+        HomogeneousAtmosphere(width=-50.)
```

Comment:
I'd add:
```python
with pytest.raises(ValueError):
    HomogeneousAtmosphere(sigma_s=0., sigma_a=0.)
```

because the albedo then is undefined.

Reply:
See my previous comment: this is unlikely to happen and checking for this in a way that makes sense, _i.e._ will really catch errors (say when you have non-uniform spectra with a few zeros in them), is quite challenging to implement.